src/app/Views/about.html

```diff
@@ -29,12 +29,12 @@ <h4>Kahla {{checkService.version}}</h4>
         <div class="text" i18n="@@CheckingForUpdates">Checking for updates...</div>
     </div>
 </ng-template>
-<div class="menu-button">
+<a class="menu-button" href="https://github.com/AiursoftWeb/Kahla.App/commits/master" target="_blank" rel="noopener noreferrer">

```

Comment:
It would be better to display commits of the tagged release.

Reply:
So how to do that?

Reply:
What about <https://github.com/AiursoftWeb/Kahla.App/tags>?
Or we can write another file `UPGRADING` like [php upgrading history](https://github.com/php/php-src/blob/master/UPGRADING)

Reply:
Good idea!